app/src/main/java/com/example/easycloset/ParseApplication.java

```diff
@@ -0,0 +1,20 @@
+package com.example.easycloset;
+
+import android.app.Application;
+
+import com.parse.Parse;
+import com.parse.ParseObject;
+
+public class ParseApplication extends Application {
+    @Override
+    public void onCreate() {
+        super.onCreate();
+        ParseObject.registerSubclass(Item.class);
+        Parse.initialize(new Parse.Configuration.Builder(this)
+                .applicationId("Ip3Oh4XMzgSUw2HLU5ZcpIBUgoCJU7MXlv8z09qY")
```

Comment:
it is fine for now, but it is not good idea to hardcode and push app id and API keys to public github repo.

Reply:
okay noted